.github/workflows/ci-deb-packages-v2.yml

```diff
@@ -245,6 +245,14 @@ jobs:
         if-no-files-found: warn
 
     - name: Copy git hash
+      if: >-
+        ${{
+          github.event_name == 'pull_request' &&
+          contains(github.event.pull_request.body, ':upload-artifacts') ||
```

Comment:
If we ever use this feature it will be in the `title` not in the `body`

Reply:
Just for the context, common practice is to put magic CI flags into body not title, this practice exists for a reason, just think how pretty title would look like when you have more than flag.

Reply:
Changed